types/index.d.ts

```diff
@@ -489,15 +491,23 @@ declare namespace Omise {
 
     interface ISchedule extends IBaseResponse {
       status: string;
+      active: boolean;
       every: number;
       period: string;
       on: Ion;
       in_words: string;
-      start_date: string;
-      end_date: string;
+      start_on: string;
+      end_on: string;
       occurrences: IOccurrences;
-      next_occurrence_dates: string[];
+      next_occurrences_on: string[];
       charge?: ICharge;
```

Comment:
Thanks for your PR @cpv123. Unfortunately, this type definitions is intended for API version `2017-11-02` and below.

It uses old style convention e.g. `start_date` and `created` (not `start_on` and `created_at` like version `2019-05-29`. 

So, could you help update these to match `2017` version. https://www.omise.co/schedules-api?version=2017-11-02

Then I can merge and release it. Thanks 👍

Reply:
@rezigned done 👍